docs/modules/ROOT/pages/how-tos/exoscale/remove_storage_node.adoc

```diff
@@ -0,0 +1,243 @@
+= Remove a storage node
+
+:kubectl_extra_args: --as=cluster-admin
+:delabel_app_nodes: yes
+:argo_app: rook-ceph
+
+[abstract]
+--
+Steps to remove a storage node of an OpenShift 4 cluster on https://www.exoscale.com[Exoscale].
+--
+
+== Starting situation
+
+* You already have a OpenShift 4 cluster on Exoscale
+* You have admin-level access to the cluster
+* You want to remove an existing storage node in the cluster
+
+== Prerequisites
+
+The following CLI utilities need to be available locally:
+
+* `docker`
+* `curl`
+* `kubectl`
+* `oc`
+* `exo` >= v1.28.0 https://community.exoscale.com/documentation/tools/exoscale-command-line-interface[Exoscale CLI]
+* `vault` https://www.vaultproject.io/docs/commands[Vault CLI]
+* `commodore`, see https://syn.tools/commodore/running-commodore.html[Running Commodore]
+* `jq`
+* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor] (version 4 or higher)
+
+== Prepare local environment
+
+. Create local directory to work in
++
+[TIP]
+====
+We strongly recommend creating an empty directory, unless you already have a work directory for the cluster you're about to work on.
+This guide will run Commodore in the directory created in this step.
+====
++
+[source,bash]
+----
+export WORK_DIR=/path/to/work/dir
+mkdir -p "${WORK_DIR}"
+pushd "${WORK_DIR}"
+----
+
+. Configure API access
++
+include::partial$exoscale/environment-vars.adoc[]
++
+include::partial$vshn-input.adoc[]
+
+. Get required tokens from Vault
++
+include::partial$connect-to-vault.adoc[]
++
+include::partial$get-hieradata-token-from-vault.adoc[]
+
+. Compile the catalog for the cluster.
+Having the catalog available locally enables us to run Terraform for the cluster to make any required changes.
++
+[source,bash]
+----
+commodore catalog compile "${CLUSTER_ID}"
+----
+
+== Set alert silence
+
+:duration: +60 minutes
+include::partial$create-alertmanager-silence.adoc[]
```

Comment:
From what I can see, this whole section is identical between this how-to, adding a node and replacing a node. Probably makes sense to move this to a partial, as any changes here will need to be reflected in all three howtos anyway.